app/frontend/src/features/messages/surfing/HostRequestView.tsx

```diff
@@ -1,73 +1,183 @@
+    {
+      onSuccess: () => {
+        queryClient.invalidateQueries(["hostRequestMessages", hostRequestId]);
+        queryClient.invalidateQueries(["hostRequests"]);
+      },
+    }
+  );
+  const respondMutation = useMutation<
+    void,
+    GrpcError,
+    Required<RespondHostRequestReq.AsObject>
+  >(
+    (req) =>
+      service.requests.respondHostRequest(
+        req.hostRequestId,
+        req.status,
+        req.text
+      ),
     {
       onSuccess: () => {
         queryClient.invalidateQueries([
-          "hostRequestMessages",
-          hostRequest.hostRequestId,
+          "hostRequest",
+          hostRequest?.hostRequestId,
         ]);
+        queryClient.invalidateQueries(["hostRequestMessages", hostRequestId]);
         queryClient.invalidateQueries(["hostRequests"]);
       },
     }
   );
 
-  const classes = useStyles();
-  return (
+  const history = useHistory();
+
+  const handleBack = () => history.goBack();
+
+  return !hostRequestId ? (
+    <Alert severity={"error"}>Invalid host request id.</Alert>
+  ) : (
     <Box className={classes.root}>
-      <Typography variant="h3">
-        {surferLoading || hostLoading ? <Skeleton /> : title}
-      </Typography>
-      {mutation.error && <Alert severity={"error"}>{mutation.error}</Alert>}
-      {isLoading ? (
+      <Box className={classes.header}>
+        <HeaderButton onClick={handleBack} aria-label="Back">
+          <BackIcon />
+        </HeaderButton>
+
+        <PageTitle className={classes.title}>
+          {!title || hostRequestError ? <Skeleton width="100" /> : title}
+        </PageTitle>
+
+        <HeaderButton
+          onClick={handleClick}
+          aria-label="Menu"
+          aria-haspopup="true"
+          aria-controls="more-menu"
+          innerRef={menuAnchor}
+        >
+          <OverflowMenuIcon />
+        </HeaderButton>
+        <Menu
+          id="more-menu"
+          anchorEl={menuAnchor.current}
+          keepMounted
+          open={menuOpen}
+          onClose={handleClose}
+        >
+          <MenuItem onClick={() => null}>Placeholder</MenuItem>
+        </Menu>
+      </Box>
+
+      <UserSummary user={otherUser} />
+
+      <Divider />
+
+      {(respondMutation.error || sendMutation.error || hostRequestError) && (
+        <Alert severity={"error"}>
+          {respondMutation.error?.message ||
+            sendMutation.error?.message ||
+            hostRequestError?.message}
+        </Alert>
+      )}
+      {isMessagesLoading ? (
         <CircularProgress />
       ) : (
         <>
-          {error && <Alert severity={"error"}>{error.message}</Alert>}
-          {messages && (
+          {messagesError && (
+            <Alert severity={"error"}>{messagesError.message}</Alert>
+          )}
+          {messages && hostRequest && (
             <>
               <MessageList messages={messages} />
-              <SendField sendMutation={mutation} />
+              <HostRequestSendField
```

Comment:
Dunno if it's related to the `TextField` change, but now this isn't showing on full width anymore and it's floated to the left 🤔

Reply:
Oops